src/main/scala/is/hail/annotations/BroadcastValue.scala

```diff
@@ -6,4 +6,14 @@ import org.apache.spark.broadcast.Broadcast
 
 case class BroadcastValue(value: Annotation, t: Type, sc: SparkContext) {
   lazy val broadcast: Broadcast[Annotation] = sc.broadcast(value)
+
+  lazy val regionValue: RegionValue = {
```

Comment:
This is going to cause a problem with @danking's coming changes because you won't know when to free the region.  I think we should just have a annotation-to-RV utility function to do this.

Reply:
This is actually more-or-less how I was planning to approach updating this class.

Latest thinking is to add:

```
def use[T](f: RegionValue => T): T =
  using(Region()) { r =>
    ...
   f(rv)
  }
```

and/or

```
def regionValue(r: Region): RegionValue = ...
```

The former is useful in non-distributed settings. The latter should cover the case where someone has designated a region for your use and you need to deserialize the globals / sample-annotations.

Reply:
OK, if @danking is happy with this, so am I.